hardware/raspberrypi/spi/README.md

```diff
@@ -136,6 +151,23 @@ When asking for say 24 MHz, the actual speed will be 15.6 MHz.
 
 Forum post: [SPI has more speeds](https://www.raspberrypi.org/forums/viewtopic.php?f=44&t=43442&p=347073)
 
+### bcm2835 library
+
```

Comment:
Do we recommend this one or spidev?

Reply:
spidev, definitely (although it also sucks). We should remove this link and say that other hardware-direct libraries are available but not recommended.

Reply:
Done.